app/views/schools/usage/show.html.erb

```diff
@@ -25,10 +25,14 @@
   </div>
 
   <div class="row">
-    <div class="col-md-12 chart-wrapper" id="chart_wrapper_<%=@chart_config[@period]%>">
```

Comment:
We've lost the `col-md-12` styling here as well as the chart-wrapper is that not needed?

Reply:
:+1: will add back in